addons/website_sale/views/templates.xml

```diff
@@ -1357,6 +1357,15 @@
                                         </div>
                                         <div id="product_option_block" class="d-flex flex-wrap w-100"/>
                                     </div>
+                                    <t
```

Comment:
Thanks for the fix! As mentioned on Discord, could you instead modify the jQuery selector in the JS code (i.e. use `closest` instead of `find` to get the `.o_product_tags` element). Doing this will allow you to keep the layout unchanged and target 18.0 :slightly_smiling_face:

Reply:
Done, thanks! I added a little bit of CSS to adjust the spacing. :)